Sources/Fuzzilli/Fuzzer.swift

```diff
@@ -93,6 +93,21 @@ public class Fuzzer {
     /// Fuzzer instances can be looked up from a dispatch queue through this key. See below.
     private static let dispatchQueueKey = DispatchSpecificKey<Fuzzer>()
 
+    /// List of generators that generate basic types
+    public let programPrefixGenerators: [CodeGenerator] = [
+            CodeGenerators.get("IntegerGenerator"),
+            CodeGenerators.get("StringGenerator"),
+            CodeGenerators.get("BuiltinGenerator"),
+            CodeGenerators.get("RegExpGenerator"),
+            CodeGenerators.get("BigIntGenerator"),
+            CodeGenerators.get("FloatGenerator"),
+            CodeGenerators.get("FloatArrayGenerator"),
+            CodeGenerators.get("IntArrayGenerator"),
+            CodeGenerators.get("TypedArrayGenerator"),
+            CodeGenerators.get("ObjectArrayGenerator"),
+            CodeGenerators.get("ObjectArrayGenerator"),
```

Comment:
Duplicate entry

Reply:
Oh Interesting, so the [existing code](https://github.com/googleprojectzero/fuzzilli/blob/4219480e371bd86a4eb5c2810e3ca37adc1a0a69/Sources/Fuzzilli/Core/FuzzEngine.swift#L73) had two entries (for a different generator) and I assumed this was intentional perhaps as to increase probability of objects being created?

Reply:
Hmm right, yeah it was probably meant for that. I think with this PR, the `fuzzer.programPrefixGenerators` now has two somewhat distinct purposes: (1) a list of CodeGenerators from which to build program prefixes, and (2) a list of CodeGenerators that can be run when no other variables are currently visible. The two sets necessarily identical (e.g. for a prefix, we could also run a CodeGenerator that requires inputs).

Maybe we should just rename this to something like 'trivialCodeGenerators` with a comment that states that these are CodeGenerators that don't require inputs and generate simple objects/values that can subsequently be used. Then we just use that list to also generate the program prefix, and maybe do that by randomly picking 5 (which should be a constant `programPrefixSize` in FuzzEngine probably) CodeGenerators from the list. WDYT?

Reply:
yeah that works for me, so essentially we still have one list then for both the ProgramBuilder and FuzzEngine to use. Do you also want to include generators that require inputs to the FuzzEngine in the way the exsiting code does it?
If we decide to go with randomly picking and running generators (rather than executing them sequentially like the existing code) when generating a prefix, then we should avoid adding generators that require inputs because running them without any available vars would cause `randVar` to crash at runtime.

Reply:
I think it's good the way it is now, i.e. without CodeGenerators that take inputs, to keep it `trivial` :)